release/src/module_dependencies/main.py

```diff
@@ -197,4 +198,55 @@ def getImmediateDependents(moduleNameList, moduleDetailsJSON):
                     
     return moduleDetailsJSON
 
+# Updates the stdlib dashboard in README.md
+def updateStdlibDashboard(moduleDetailsJSON):
+    try:
+        readMeFile = urlOpenWithRetry("https://raw.githubusercontent.com/ballerina-platform/ballerina-standard-library/main/README.md")
+    except:
+        print('Failed to read README.md file')
+        sys.exit()
+
+    updatedReadMeFile = ''
+
+    for line in readMeFile:
+        processedLine = line.decode("utf-8")
+        updatedReadMeFile += processedLine
+        if "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|" in processedLine:
+            break
+
+    # Modules in levels 0 and 1 are categorized under level 1
+    # A single row in the table is created for each module in the module list    
+    levelColumn = 1
+    tempLevel = 0
+    for module in moduleDetailsJSON['modules']:
+        if tempLevel != module['level'] and module['level'] != 1:
+            levelColumn = module['level']
+            tempLevel = module['level']
+
+        row = ("|" + str(levelColumn) + "|[" + module['name'].split('-')[-1] + "](https://github.com/ballerina-platform/" 
```

Comment:
We are going to print the level in each line here, right? That's not the way it is now.

We can overcome this by using a variable here and setting it on the condition above (Whether we are at a new level or not).

Reply:
No this doesn't update the level on each row. I'm setting the level to an empty string at the end of the loop